samples/e2eTestUtils/ElectronPlaywrightTestUtils.ts

```diff
@@ -112,3 +112,16 @@ export function createFolder(foldername: string) {
         fs.mkdirSync(foldername, { recursive: true });
     }
 }
+
+export async function getAuthCodeUrl(page: Page): Promise<string> {
```

Comment:
let's consider renaming this function. something like "retrieveAuthCodeUrlFromBrowserContext" would better at signaling what it does

Reply:
done